examples/reference/elements/bokeh/Scatter.ipynb

```diff
@@ -56,8 +58,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Scatter (color='k' marker='s' size=10)\n",
-    "hv.Scatter(coords)[0:12] + hv.Scatter(coords)[12:20]"
+    "layout = hv.Scatter(coords)[0:12] + hv.Scatter(coords)[12:20]\n",
+    "layout.opts(opts.Scatter(color='k', marker='s', size=10))"
```

Comment:
Again, would reuse the styled scatter from above, to avoid styling here altogether.